### PR TITLE
Remove references to decision validation from readme

### PR DESCRIPTION
**Pull request by elpoelma**
## Overview
Currently, this package does not provide any validation options for decision nodes. This PR removes all references to decision validation from the README file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,8 +247,7 @@ The options for mode are:
 ### Besluit 
  :warning: The besluit plugin is incompatible with the `regulatoryStatement` mode of  [article-structure](#Article Structure).
 
-This will add needed rdfa structures to create a besluit and add some validation. There is no direct interaction with the plugin. The validation will pop up in the sidebar. You can try this out by deleting the title and seeing the following pop up:
-![besluit plugin](https://imgur.com/iwCudJy.png)
+This will add needed rdfa structures to create a besluit.
```
Comment (x-m-el) [inline README.md:250]:
```suggestion
This will add rdfa structures needed to create a besluit.
```

**Review by x-m-el — APPROVED** [review]
I do wonder:
Can't we just remove the besluit plugin from embeddable in this case? 
article-structures also adds the neccesary rdfa structures, together with buttons to actually insert them. There is no need for this plugin anymore then.

but maybe that's a plan for later :shrug:

File: README.md (hunk 1 above)
Comment (x-m-el) [reply README.md:250]: removing "needed" could also be better. It is a bit redundant.

**Comment by elpoelma** [comment]
> I do wonder: Can't we just remove the besluit plugin from embeddable in this case? article-structures also adds the neccesary rdfa structures, together with buttons to actually insert them. There is no need for this plugin anymore then.
> 
> but maybe that's a plan for later 🤷

I think the `besluit` plugin still adds some prosemirror (like motivation etc.).

**Pull request merged**